wren-ui/src/apollo/server/schema.ts

```diff
@@ -757,6 +785,10 @@ export const typeDefs = gql`
       threadId: Int!
       data: CreateThreadResponseInput!
     ): ThreadResponse!
+    createRegeneratedThreadResponse(
```

Comment:
If the purpose of this mutation is to generate corrections, maybe name it `createCorrections` ?